website/docs/tutorials/07-testing/02-cheatcodes/example.md

```diff
@@ -18,6 +18,9 @@ This cheatcode is only available in [setup cases](../README.md#setup-case).
 :::warning
 `example` is not limited by [`max_examples`](./max-examples.md) and is not connected to it in any way.
 :::
+:::warning
```

Comment:
We have to update fuzzing tutorial, default fuzzing is mentioned there in many places

Reply:
[update the fuzzing tutorial](https://github.com/software-mansion/protostar/pull/995/commits/cf20cf1d265a8ff397e6487a07546b7b2946fab2) I tried to cover this but might've missed something, let me know, thanks!